docs/manuals/platform-installation-manual.md

```diff
@@ -0,0 +1,44 @@
+# ZGW Office Add-In Componenten Installatie
+
+De ZGW Office Add-In bestaat uit verschillende componenten die samen zorgen voor de integratie met Microsoft Word en de 
+ZGW APIs. Deze componenten moeten correct worden geïnstalleerd en geconfigureerd om de add-in goed te laten 
+functioneren.
+
+## Componenten overzicht
+De ZGW Office Add-In bestaat uit de volgende componenten:
+- **ZGW Office Add-In**: De add-in die geïnstalleerd is in de Microsoft Word desktop en webversie. Deze add-in is beschikbaar voor installatie vanuit de ZGW Office Frontend Proxy via een `metadata.xml` bestand.
+- **ZGW Office Frontend Proxy**: Een proxy service die de communicatie tussen de ZGW Office Add-In en de ZGW Office Backend Service mogelijk maakt. Deze service is verantwoordelijk voor het doorsturen en valideren van verzoeken van de add-in.
+- **ZGW Office Backend Service**: De backend service die de communicatie met de ZGW APIs mogelijk maakt. Deze service is verantwoordelijk voor verdere koppelingen binnen het Applications Platform waarin de ZGW APIs en Objecten API draaien.
+
+Zie voor meer informatie de [C4 Context Diagram](https://c4model.com/diagrams/system-context) documentatie.
+
+## Voorbereiding
+Voordat u begint met de installatie, zorg ervoor dat u de volgende stappen hebt voltooid:
```

Comment:
For the ZAC manuals we mostly use the informal term `je` (but we are not 100% consistent I noticed). I would propose to do that here too. `u` sounds very formal and it's better to be a bit consistent here I feel.